zombie_scout.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |s|
   s.add_dependency('parser', '~> 2.1')
   s.add_dependency('thor', '~> 0.18')
   s.add_dependency('flog', '~> 4.2')
-  s.add_development_dependency "bundler", "~> 1.10"
+  s.add_development_dependency "bundler", "> 1.10"
```

Comment:
Running `gem build zombie_scout.gemspec` gives me this warning:

```
WARNING:  open-ended dependency on bundler (> 1.10, development) is not recommended
  if bundler is semantically versioned, use:
    add_development_dependency 'bundler', '~> 1.10', '> 1.10'
```

What were you going for with this part of the change?

Since this is a dev dependency, is there any good reason to not just bump this to `~> 2.0`?

Reply:
you are right, I should fix this, but I think we need to keep compatibility with 1.x because of old rubies still supported